mne/preprocessing/ica.py

```diff
@@ -2302,9 +2306,9 @@ def _plot_corrmap(data, subjs, indices, ch_type, ica, label, show, outlines,
 
 
 @verbose
-def corrmap(icas, template, threshold="auto", label=None,
-            ch_type="eeg", plot=True, show=True, verbose=None, outlines='head',
-            layout=None, sensors=True, contours=6, cmap='RdBu_r'):
+def corrmap(icas, template, threshold="auto", label=None, ch_type="eeg",
+            plot=True, show=True, verbose=None, outlines='head', layout=None,
+            sensors=True, contours=6, cmap='RdBu_r'):
```

Comment:
So cmap should default to None, and if it is None, is set in `_plot_corrmap` to `reds_r` if `ch_type` is grads, and otherwise to `RdBu_r`? @Eric89GXL you added this code, can you check if that's appropriate?

(I'm not experienced with MEG yet ...)